src/pykeen/utils.py

```diff
@@ -1679,6 +1688,9 @@ def merge_kwargs(kwargs: OptionalKwargs, **extra_kwargs: Any | None) -> Optional
     :return:
         Updated parameters.
     """
+    if isinstance(kwargs, Sequence):
```

Comment:
this might not work as intended, doesn't a dict also count as a sequence?

Reply:
Does not look like it is.
```python-console
>>> from collections.abc import Sequence
>>> isinstance({"a": 1}, Sequence)
False
```
I think it is only a collection (i.e. no order guarantee, although the implementation does guarantee that since some Py version).

Reply:
okay, thanks for clarifying!